Python/Password_generator_App.py

```diff
@@ -0,0 +1,45 @@
+import string
+import random
+import tkinter as tk
+from tkinter import messagebox
+# from PIL import ImageTk
+root = tk.Tk()
+root.geometry('350x260')
+root.title("Password Generator App")
+root.resizable(False,False)
+# root.bg = ImageTk.PhotoImage(file="14.jpg")
+# root.bg_image=tk.Label(root,image=root.bg).place(x=0,y=0)
+x=tk.StringVar()
+s = []
+
+s1 = string.ascii_uppercase
+s2 = string.ascii_lowercase
+s3 = string.digits
+s4 = string.punctuation
+def generatepass():
+    a = x.get()
+    try:
+        if a == "":
+            messagebox.showerror("Error","Invalid Input",parent=root )
+        passlen = int(a)
+        if passlen <= 0 :
+            messagebox.showerror("Error","Invalid Input",parent=root )
+        else:
+            s.extend(list(s1))
+            s.extend(list(s2))
+            s.extend(list(s3))
+            s.extend(list(s4))
+            random.shuffle(s)
+            pas = "".join(s[0:passlen])
+            lbl = tk.Label(root,text=pas,font=("times new roman",15,"bold")).place(x=50,y=170,width=250)
+    except:
+        pass
+def finish():
+    exit()
+
+labl = tk.Label(root,text="Enter the Length of the Password :-",font=("times new roman",15,"bold")).place(x=10,y=20)
```

Comment:
Just refactor these lines, all in the same line doesn't look nice. I am looking for something similar to this:
```
tk.label(root,
       text = "fdf"
       font = 'df'   
```
Like this

Reply:
ok i will do it!!!